Orange/data/io.py

```diff
@@ -5,13 +5,14 @@
 from itertools import chain
 
 import bottlechest as bn
-import numpy as np
+from sklearn import tree
 from scipy import sparse
 # We are not loading openpyxl here since it takes some time
 
 from Orange.data import _io
```

Comment:
Seeing where `_io` is used, wouldn't it make sense to just move the exotic import into that function? For a faster boot time at the least! :smiley: